index.js

```diff
@@ -223,26 +185,48 @@ const HOME_MENU_BUTTONS_ARRAY = [
 //
 
 const renderHomeMenu = () => {
+    const screenWrapperElement = getScreenWrapperElement()
+
+    if (screenWrapperElement) {
+        screenWrapperElement.remove()
+    }
+
     const mainElement = createElement("main", undefined, appElement)
 
     createElement("p", {
         innerText: "HOME"
     }, mainElement)
 
-    HOME_MENU_BUTTONS_ARRAY.forEach(({ id, innerText, options, param2, onclick }, index) => createElement("button", {
+    HOME_MENU_BUTTONS_ARRAY.forEach(({ id, innerText, options, buttonName, onclick }, index) => createElement("button", {
         id,
         className: "home-options",
         innerText,
         onclick: () => {
 
             if (onclick) {
-                onclick()
-            }
+                removeSubMenuElement()
```

Comment:
This is if all incorrect

Reply:
Don't know why you still seeing that, I've changed that already. Anyway